src/less/toggle-button/toggle-button.less

```diff
@@ -34,6 +34,10 @@
     }
 }
 
+.toggle-button * {
```

Comment:
@agliga , @LuLaValva , this will be critical for completing the ebay ui piece in the most optimal way using delegated event handlers. Elements with `pointer-events` inside `button` will actually "steal" the event target. This ensures a smooth detection of the button as the `target` in delegated event handlers for `toggle-button-group`.